src/pages/resources/customer-dashboard/lesson2.md

```diff
@@ -14,7 +14,7 @@ Within the newly created app, you have seen the `.env` file which contains your
 
 Firstly, `package.json` is the [crucial part](https://docs.npmjs.com/creating-a-package-json-file) of almost every NodeJS project. It contains the list of dependencies, version, reproducible builds, etc.
 
-Then `manifest.yml` is the cockpit of your Firefly app backend. It lists the declaration of serverless actions including name, source files, runtime kind, default params, annotations, and so on. You can find the grammar of writing manifest [here](https://github.com/apache/openwhisk-wskdeploy/blob/master/docs/programming_guide.md#wskdeploy-utility-by-example).
+Then `app.config.yaml` is the cockpit of your Firefly app backend. It lists the declaration of serverless actions including name, source files, runtime kind, default params, annotations, and so on. You can find the grammar of writing manifest [here](https://github.com/apache/openwhisk-wskdeploy/blob/master/docs/programming_guide.md#wskdeploy-utility-by-example).
```

Comment:
For ExC app, the manifest info now moves to ext.config.yaml inside the extension folder, not app.